termwiz/Cargo.toml

```diff
@@ -32,6 +32,7 @@ serde = {version="1.0", features = ["rc", "derive"], optional=true}
 sha2 = "0.9"
 terminfo = "0.7"
 thiserror = "1.0"
+tmux-cc = {version = "0.1", path = "../tmux-cc"}
```

Comment:
dependency management thinking: `termwiz` is its own crate on crates.io.
Adding a dep here means that we'll also either:

* need to publish `tmux-cc` to crates.io (and I'm not sure that that makes sense as it doesn't seem obviously reusable)
* move tmux-cc to be a module inside termwiz, and potentially make it an optional feature

Reply:
Woa, didn't expect that. Move into `termwiz` seems more reasonable, maybe better to move it after all other issues fixed?